src/figuresClasses.ts

```diff
@@ -1,19 +1,80 @@
 export interface Figure {
+  shape: string;
+  color: string;
 
+  getArea(): number;
 }
 
 export class Triangle implements Figure {
+  shape: string = 'triangle';
```

Comment:
let's create a type for shape - either union or enum and use it instead of string to restrict expected values.

Reply:
OK, thanks for your tip)